OrbitVulkanLayer/DeviceManager.h

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_VULKAN_LAYER_PHYSICAL_DEVICE_MANAGER_H_
```

Comment:
Change due to name change of file / class.

Reply:
Done, thanks!